blackbox/docs/general/information-schema.rst

```diff
@@ -236,6 +236,14 @@ options of all available views.
 | ``is_updatable``    | Whether the view is updatable. Not applicable for CrateDB, always returns ``FALSE`` | ``Boolean`` |
 +---------------------+-------------------------------------------------------------------------------------+-------------+
 
+.. note::
+
+   If you drop the table of a view, the view will not show up in the
```

Comment:
It sounds quite weird behaviour for a user? Why was it implemented like that?

Reply:
If the table is missing we cannot analyze the query anymore. We'll probably implement a cascading delete later on or prevent deletions if a table is used in a view.

Reply:
I understand about not implementing the cascading, but not showing the view in the info_schema tables while it still exists (even though erroneous) seems quite weird to me as user experience.